code/modules/projectiles/updated_projectiles/magazines/rifles.dm

```diff
@@ -17,20 +17,20 @@
 /obj/item/ammo_magazine/rifle/extended
 	name = "\improper M41A extended magazine (10x24mm)"
 	desc = "A 10mm assault extended rifle magazine."
-	icon_state = "m41a_ext"
+	icon_state = "m41a1_ext"

```

Comment:
The name of the gun is `M41A`, why is it being changed to `M41A1` in the icon?

Reply:
It was renamed in #1494 not sure why it doesn’t show up here.

Reply:
Because you haven't rebased likely

Reply:
i renamed the standard pulse rifle to m41a1, the og is called m41a. everything is correct here